compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2185,6 +2174,8 @@ impl CheckAttrVisitor<'_> {
                 "attribute `{}` without any lints has no effect",
                 attr.name_or_empty()
             )
+        } else if attr.name_or_empty() == sym::default_method_body_is_const {
+            format!("`default_method_body_is_const` has been removed")
```

Comment:
```suggestion
            format!("`default_method_body_is_const` has been replaced with `#[const_trait]` on traits")
```

Reply:
This was marked resolved but was not changed.

Reply:
Oops thanks! I did apply it and screw up a rebase